manta-pay/abstract-protocol.tex

```diff
@@ -47,6 +75,7 @@ \subsection{Abstract Cryptographic Schemes}
         \SecretKey    &: \Type \\
         \PublicKey    &: \Type \\
         \SharedSecret &: \Type \\
+        \SecretKeyDistribution &: \mathfrak{D}\bra{\SecretKey} \\
```

Comment:
$\mathfrak{D}$ is not used in Definition 4.1.5 and the following properties.

Reply:
@SupremoUGH @bhgomes

Reply:
It's used later on, e.g. when constructing a sender according to the guidelines you have so sample an ephemeral secret key from the SecretKeyDistribution, so it must be specified beforehand (and then, why not on the dfn?)